src/client.ts

```diff
@@ -638,61 +608,45 @@ export class Client {
    * @category Lattice
    * @returns The decrypted response.
    */
-  public removeAbiRecords (opts: { sigs: (number | string)[] }, _cb?: (err?: string, data?: {
-    numRemoved: number;
-    numTried: number;
-  }) => void, cbData = { numRemoved: 0, numTried: 0 }): Promise<{
-    err?: string, data?: {
-      numRemoved: number;
-      numTried: number;
-    }
-  }> {
+  public removeDecoders (opts: { decoderType: number, decoders?: Buffer[], rmAll?: boolean },
```

Comment:
The comment docs for this function weren't updated when the name changed.

Reply:
[756bc00](https://github.com/GridPlus/gridplus-sdk/pull/353/commits/756bc001e62e4ba9fce75734645727253137ddc2)